src/net/RTCP/RTCPCompoundPacket.cs

```diff
@@ -123,6 +133,27 @@ public RTCPCompoundPacket(byte[] packet)
             }
         }
 
+        public static string HexDump(byte[] bytes)
+        {
+            if (bytes == null) { return "<null>"; }
+
+            var sb = new StringBuilder();
+            for (int i = 0; i < bytes.Length; i++)
+            {
+                // Start a new line every 16 bytes.
+                if (i % 16 == 0)
+                {
+                    if (i > 0)
+                    {
+                        sb.AppendLine();
+                    }
+                    sb.Append($"{i:X4}: ");
+                }
+                sb.Append($"{bytes[i]:X2} ");

```

Comment:
This is very expensive in older targets.

Use this instead:

```suggestion
                sb.Append(bytes[i]("X2")).Append(' ');
```